graal-js/src/com.oracle.truffle.js/src/com/oracle/truffle/js/nodes/promise/BuiltinPromiseRejectionTracker.java

```diff
@@ -119,12 +122,12 @@ public void promiseReactionJobsProcessed() {
             }
             info.warned = true;
             if (mode == JSContextOptions.UnhandledRejectionsTrackingMode.HANDLER) {
-                Object handler = JSRealm.get(null).getUnhandledPromiseRejectionHandler();
+                Object handler = realm.getUnhandledPromiseRejectionHandler();
                 if (handler != null) {
-                    JSInteropUtil.call(handler, new Object[] {info.reason, unhandled});
+                    JSRuntime.call(handler, Undefined.instance, new Object[] {info.reason, unhandled});
```

Comment:
also switched this to `JSRuntime` as it seemed to have more optimal paths for calling JS functions directly, and falls back to `JSInteropUtil` anyways.